packages/scss/src/components/_button.scss

```
@@ -34,11 +34,29 @@
 	}
 }
 
-@mixin buttonLinkColoring($palette) {
+@mixin buttonLinkColoring($palette, $activeShadowPalette ) {
+	color: _color($palette, '700');
+
+	&:hover:not([disabled]) {
+		background-color: _color($palette, "50");
```

```suggestion
		background-color: _color($palette, "50");
		box-shadow: none;
```

Car sinon il prend le box-shadow par défaut du button:hover